nibabel/parrec.py

```diff
@@ -990,6 +1006,62 @@ def get_rec_shape(self):
         inplane_shape = tuple(self._get_unique_image_prop('recon resolution'))
         return inplane_shape + (len(self.image_defs),)
 
+    def _strict_sort_keys(self):
+        """ Determine the sort order based on several image definition fields.
+
+        If the sort keys are not unique for each volume, we calculate a volume
+        number by looking for repeating slice numbers
+        (see :func:`vol_numbers`).  This may occur for diffusion scans from
+        older V4 .PAR format, where diffusion direction info is not stored.
+
+        Data sorting is done in two stages:
+            - run an initial sort using several keys of interest
+            - call `vol_is_full` to identify potentially missing volumes
+              and add the result to the list of sort keys
+        """
+        # Sort based on a larger number of keys.  This is more complicated
+        # but works for .PAR files that get missorted by the above method
```

Comment:
I guess it's not "the above method" now.

Maybe put the other sorting method into a `lax_sort` method, and refer to that?